### PR TITLE
DW-14:  Styles header

### PR DESCRIPTION
**Pull request by LautaroTinelli**
# [DW-14: Add styles to Header](https://trello.com/c/q8htUtXn/117-dw-14-add-styles-to-header)

## Type of change

> Exchange rate entered. The possible values ​​for this section are:
>
> - Feature (new feature of the server/application)

## Description

> - Add icons and center phone number and times.
> - Add styles to navbar.

## Attachments

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,71 +22,75 @@
 
   <body>
     <header>
+      <div class="topbar">
+          <p><i class="fas fa-mobile-button icon"></i>+1 5493 41123 45</p>
+          <p><i class="fas fa-clock icon"></i>Every day: 11AM - 23PM</p>
+      </div>
       <div>
```
Comment (maxidelfino) [inline index.html:29]: Remove this <div> it is unnecessary

File: src/styles/index.css
Comment (maxidelfino) [inline src/styles/index.css:1]: Check the typography of the letters. Must match the figma.
image
image

File: src/pages/about-us/index.html [hunk 2]
```diff
@@ -20,20 +22,31 @@
...
-          <li><a href="./index.html">About Us</a></li>
-          <li><a href="../menu-specials/index.html">Menu & Specials</a></li>
-          <li><a href="../events-contacts/index.html">Events & Contacts</a></li>
-          <li><a href="../gallery/index.html">Gallery</a></li>
-        </ul>
+      <div class="topbar">
+        <p><i class="fas fa-mobile-button icon"></i>+1 5493 41123 45</p>
+        <p><i class="fas fa-clock icon"></i>Every day: 11AM - 23PM</p>
+    </div>
+    <div>
```
Comment (maxidelfino) [inline src/pages/about-us/index.html:29]: Remove this <div> it is unnecessary

File: src/styles/index.css
Comment (maxidelfino) [inline src/styles/index.css:1]: On mobile phones and tablets, the background color of the links must be respected!
image
image

Comment (maxidelfino) [inline src/styles/index.css:1]: Remove border to toggler button

Comment (maxidelfino) [inline src/styles/index.css:1]: Delete background-color of icons
image
image

File: src/styles/index.css [hunk 3]
```diff
@@ -1,8 +1,62 @@
...
+  font-size: 16px;
+  color: #ffffff;
+  text-align: center;
+  text-decoration: none;
+  white-space: nowrap;
+}
+
+#links .active {
+  color: #d9ba85;
+  border-bottom: 2px solid #d9ba85;
```
Comment (maxidelfino) [inline src/styles/index.css:55]: Add   `transition: all 0.5s;`  to maintain consistency

**Review by maxidelfino — CHANGES_REQUESTED** [review]

File: src/styles/index.css
Comment (LautaroTinelli) [reply src/styles/index.css:1]: Done

Comment (LautaroTinelli) [reply src/styles/index.css:1]: Done

**Review by maxidelfino — APPROVED** [review]

**Review by TomiJortiz — APPROVED** [review]

**Pull request merged**